requirements.txt

```diff
@@ -23,3 +23,6 @@ pydocstyle~=1.0.0
 cmakelint~=1.3.4
 requests_mock~=0.7.0
 vim-vint~=0.3.6
+# Stable version 3.2 is problematic in Windows.
+# Use >=3.3 once nltk-3.3 is released. 
+nltk==3.1
```

Comment:
Can we use ~=3.1.0 ?
They may have patched versions of 3.1 which go like 3.1.x later ?